pkg/manifests/manifests_test.go

```diff
@@ -29,30 +29,6 @@ func TestManifests(t *testing.T) {
 		},
 	}
 
-	if _, err := f.OperatorNamespace(); err != nil {
-		t.Fatal(err)
-	}
-	if _, err := f.OperatorCustomResourceDefinition(); err != nil {
-		t.Fatal(err)
-	}
-	if _, err := f.OperatorRole(); err != nil {
```

Comment:
Looks like we can keep f.OperatorRole() and f.OperatorRoleBinding()?

Reply:
We need to revisit RBAC holistically after this